app/assets/javascripts/dashboard.js

```diff
@@ -9,10 +9,13 @@ function auto_refresh(){
 
   if (element[0]) {
     refresh_timeout = setTimeout(function(){
-      if ($(".auto-refresh").hasClass("on")) {
+      var autoRefreshIsOn = $(".auto-refresh").hasClass("on");
+      var hasFocus = tfm.document.doesDocumentHasFocus();
+
+      if (autoRefreshIsOn && hasFocus) {
         Turbolinks.visit(window.location.href);
       }
-    },60000);
+    }, 60000);
```

Comment:
does it make sense to use a setting / env variable here like we have in the notifications  polling?

Reply:
I'm not sure if it makes sense or not but anyway I don't think it should be apart of this PR.